### ISSUE TITLE
Consider adding tail to moving mouse

### ISSUE THREAD

**Issue by Meai** [rrweb-io/rrweb#51]
I compared rrweb's current mouse highlight with the visually very strong animated tail from [Demo LINK: Yandex Metrica](https://metrica.yandex.com/inpage/visor-proto?visit_id=380380303560215976&date=2019-02-05&user_id_hash=2516973861&date_visit=&offset=0&id=44147844) and I think psychologically it is easier to follow the user's intent while on the site with the tail. 
I'm sure you've heard this complaint many times when you did some presentation, people always complain that you shouldn't move the mouse so fast - why? Because it's hard to follow for people, even if you enlarge the screen by a lot. 
The small circle in my opinion is not enough, the way that Yandex does it allows me to kinda see the history of the movement even if I didn't manage to follow the mouse the moment it zig zagged. However it may be very important to see these nuanced movements of the mouse, it gives you an idea of how the user felt while moving the mouse.

**Comment by Yuyz0112** [comment]
Thanks!
I have taken a look and it is implemented in canvas. A similar DEMO here: https://codepen.io/keerotic/pen/rOdgbr
